<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>202</red>
        <green>208</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>68</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>181</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>202</red>
        <green>208</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>100</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>68</red>
        <green>68</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>181</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>202</red>
        <green>208</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>181</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Create Cloth</string>
   </property>
   <property name="iconSize">
    <size>
     <width>59</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoulder to Sleeve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width Along Bottom</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Round Neck Shirt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V-Neck Shirt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hoodie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tank Top</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cloth Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Neck to Hem</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chest</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoulder to Shoulder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sleeve Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoulder to Pit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width of Neck</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Front Neck Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Back Neck Height</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>800</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>770</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>740</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>710</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>680</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>650</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>620</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>590</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="page_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>820</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pit to Sleeve</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Center Back Length</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width Bottom</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="page_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>920</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoodie Height</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoodie Width</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4"/>
  </widget>
  <widget class="QPushButton" name="btnSelectImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Load Images</string>
   </property>
  </widget>
  <widget class="QLabel" name="Selected_Image">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>No Image Slected</string>
   </property>
  </widget>
  <widget class="QLabel" name="WaitDialog">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Creating...Please Wait</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Cloth</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
